### DISCUSSION TITLE
建议加入拉取订阅源时可指定 UA 功能

### DISCUSSION THREAD

**Discussion by sumsethan** [Guovin/iptv-api#740] [Ideas]
在 subscribe.txt 中添加的某些订阅源，以浏览器 UA 访问时不会返回m3u 内容，只有以特定 UA 访问时才会返回m3u 内容
如：
curl https://tv.iill.top/m3u/Gather
返回为空
curl --user-agent "AptvPlayer/1.3.8"  https://tv.iill.top/m3u/Gather
返回为 m3u 内容

建议在 config 中加上可定义 UA 的功能及字段，以将类似 m3u 加入到筛选列表中

**Comment by Guovin** [comment]
只支持可直接获取的，暂不支持这类需要验证的订阅源。
